browser-switch/build.gradle

```diff
@@ -5,7 +5,8 @@ plugins {
 }
 
 android {
-    compileSdkVersion rootProject.compileSdkVersion
+    namespace "com.braintreepayments.api.browserswitch"
```

Comment:
random: This might be a good time to migrate the actual code packages to `com.braintreepayments.api.browserswitch`. It'll lessen the probability of collision with other libraries. Definitely work for a separate PR.

Package private has worked well for us, but it may make the Core SDK more robust if we migrate to `@RestrictTo` annotations one day. Could also be `v6` work. Bringing it up now so we don't forget.

Reply:
Discussed and have a ticket to investigate this further 👍